cypress.json

```diff
@@ -6,5 +6,7 @@
   "pluginsFile": ".cypress/plugins/index.js",
   "screenshotsFolder": ".cypress/screenshots",
   "supportFile": ".cypress/support/index.js",
-  "videosFolder": ".cypress/videos"
+  "videosFolder": ".cypress/videos",
+  "requestTimeout": 60000,
+  "responseTimeout": 180000
```

Comment:
These timeout settings will be applied to all integration test cases?
Do we expect user to wait for more than 180s?

Reply:
Yes. will be applied to all test cases, but this configuration can be overridden in single test cases. 

We don't expect user to wait for more than 180s. If response doesn't come within 180s, test will fail. 

I increased timeout to 180s is because after I change Kibana source code from ODFE Kibana to elastic/kibana, the load time somehow increases a lot, even though I am already starting kibana with OSS version via `yarn start --no-base-path --oss`. Will explore more to see how to configure in a better way.